lib/Checkout/Risk/RiskClient.php

```diff
@@ -10,6 +10,9 @@
 use Checkout\Risk\PreAuthentication\PreAuthenticationAssessmentRequest;
 use Checkout\Risk\PreCapture\PreCaptureAssessmentRequest;
 
+/**
+ * @deprecated Risk endpoints have been officially no longer supported, This module will be removed in a future release.
```

Comment:
I think that `Risk endpoints are no longer supported officially` would be better